<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addJobFile">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Job File</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>New</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ready</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Running</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Waiting</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Process</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="jobFileText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="showJobText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>191</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
